platform-dependent-components/problem-solver/cxx/generate-response-module/agent/GenerateResponseAgent.cpp

```diff
@@ -69,19 +72,21 @@ void GenerateResponseAgent::attachAnswer(
     ScAddr const & messageAddr,
     ScAddr const & answerAddr)
 {
-  utils::GenerationUtils::generateRelationBetween(&m_memoryCtx, messageAddr, messageAnswer, Keynodes::nrel_reply_structure);
-  m_memoryCtx.CreateEdge(ScType::EdgeAccessConstPosTemp, answerAddr, messageAnswer);
+  utils::GenerationUtils::generateRelationBetween(
+      &m_memoryCtx, messageAddr, messageAnswer, Keynodes::nrel_reply_structure);
+  m_memoryCtx.CreateEdge(ScType::EdgeAccessConstPosPerm, answerAddr, messageAnswer);
 }
 
-ScAddr GenerateResponseAgent::createActionNode(ScAddr const & message)
+ScAddr GenerateResponseAgent::createActionNode(ScAddr const & message, ScAddr const & conceptAddr)
```

Comment:
you don't need `conceptAddr` here in parameter, find it in `processParamsFromMessage`